tests/ui_tests/app/page/cluster_list/page.py

```diff
@@ -51,3 +58,53 @@ def get_cluster_info_from_row(self, row: int) -> dict:
             "description": self.find_child(cluster_row, row_elements.description).text,
             "state": self.find_child(cluster_row, row_elements.state).text,
         }
+
+    def click_action_btn_in_row(self, row: WebElement):
+        self.find_child(row, self.table.table.ClusterRow.actions).click()
+
+    def click_import_btn_in_row(self, row: WebElement):
+        self.find_child(row, self.table.table.ClusterRow.imports).click()
+
+    @allure.step("Run action {action_name} for cluster")
+    def run_action_in_cluster_row(self, row: WebElement, action_name: str):
+        self.click_action_btn_in_row(row)
+        self.wait_element_visible(self.table.table.ActionPopup.block)
+        self.find_and_click(self.table.table.ActionPopup.button(action_name))
+        self.wait_element_visible(ActionDialog.body)
+        self.find_and_click(ActionDialog.run)
+
+    @contextmanager
+    def wait_cluster_state_change(self, row: WebElement):
+        state_before = self.find_child(row, self.table.table.ClusterRow.state).text
+        yield
+
+        def wait_state():
+            state_after = self.find_child(row, self.table.table.ClusterRow.state).text
+            assert state_after != state_before and state_after != self.table.loading_state_text
+
+        wait_until_step_succeeds(wait_state, period=1, timeout=self.default_loc_timeout)
+
+    @allure.step("Get cluster state")
+    def get_cluster_state_from_row(self, row: WebElement):
+        return self.find_child(row, self.table.table.ClusterRow.state).text
+
+    @allure.step("Get row by cluster name '{cluster_name}'")
+    def get_row_by_cluster_name(self, cluster_name: str) -> list:
```

Comment:
I think the return type is WebElement or something like it, not list?

Reply:
thnx! changed